lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -526,13 +526,13 @@ public function notLike($x, $y)
      * Creates a CONCAT() function expression with the given arguments.
      *
      * @param mixed $x First argument to be used in CONCAT() function.
-     * @param mixed $y Second argument to be used in CONCAT() function.
+     * @param mixed $y,... Other arguments to be used in CONCAT() function.
      *
      * @return Expr\Func
      */
-    public function concat($x, $y)
+    public function concat($x)
```

Comment:
This is a BC break, and does not make sense. This function should have 2 non-optional arguments. Arguments beyond the 2 in the prototype should be used as well, but the $y should definitely stay.